PersonenOrtService/Controllers/PersonController.cs

```diff
@@ -32,5 +32,13 @@ public double PersonMeanAge()
         {
             return PersonHelper.GetMeanAge(PersonHelper.Persons);
         }
+
+
+        [HttpGet]
+        [Route("PersonNameLetterCount")]
+        public double PersonNameLetterCount()
```

Comment:
That looks really strange to me... please add comment...

Reply:
Very strange - Return is wrong and method should preferably be named after the action (`get[...]`)